nexus/tests/config.test.toml

```diff
@@ -124,6 +124,20 @@ v2p_mapping_propagation.period_secs = 30
 abandoned_vmm_reaper.period_secs = 60
 saga_recovery.period_secs = 600
 lookup_region_port.period_secs = 60
+# The purpose of the `instance-updater` background task is to ensure that update
+# sagas are always *eventually* started for instances whose database state has
+# changed, even if the update saga was not started by the Nexus replica handling
+# an update from sled-agent. This is to ensure that updates are performed even
+# in cases where a Nexus crashes or otherwise disappears between when the
+# updated VMM and migration state is written to CRDB and when the resulting
+# update saga actually starts executing. However, we would prefer update sagas
+# to be executed in a timely manner, so for integration tests, we don't want to
+# *rely* on the instance-updater background task for running these sagas.
+#
+# Therefore, set a period long enough that this task won't activate during a
+# reasonable integration test execution. Tests for the instance-updater task
+# will explictly activate it.
```

Comment:
Ah, yeah, good call. That's probably better, I'll go back and change it.

Reply:
Hmm, I'm not totally sure if this will work or not. It looks like the existing `disable` flags make _any_ activation of a background task return immediately. Here, we have both periodic activations of the `instance-updater` task *and* a bunch of explicit activations whenever we need to do an instance update and the instance is already locked. I'm. not sure whether existing tests rely on those explicit activations or not. I'm going to try it and see.

Reply:
Ah, never mind, I was wrong about that! We can just totally siable the background task: https://github.com/oxidecomputer/omicron/pull/5749/commits/ce3c35decc153198b9467f2544db801b40f3bb21